Server.py

```diff
@@ -6,20 +6,24 @@ class Server(Network):
     def __init__(self):
         super().__init__()
         self.socket.bind((self.get_ip(), self.port))
-        self.socket.settimeout(2)  # interferes with stopping
         print("Server Started")
         self.client = ("", 0)
+        self.last_sent= None
 
     def receive_data(self):
         try:
             x = self.socket.recvfrom(4096)
             self.client = x[1]
+            self.socket.settimeout(1)  # interferes with stopping on further calls
```

Comment:
@sanjit Why did we get rid of timeout in client and not server?

Reply:
Oh i see it is because this file does the timeout re-send, Can we DECREASE the timeout rate to HELP SOLVE the latency?

Reply:
let's "tune" this later after when we properly investigate latency issues